sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -697,11 +697,21 @@ sycl::detail::pi::PiProgram ProgramManager::getBuiltPIProgram(
     return Cache.getOrInsertProgram(CacheKey);
   };
 
+  if (std::getenv("SYCL_CACHE_DISABLE"))
```

Comment:
Please add docs to https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md